front/src/main.js

```diff
@@ -15,6 +16,7 @@ new Vue({
   router,
   store,
   vuetify,
+  i18n,
   created () {
     // load data from localStorage to vuex (on f5)
     const userString = localStorage.getItem('user');
```

Comment:
might, do you need INIT_LANGUAGE action in store?

Reply:
Have added this.$store.dispatch('locale/setLanguage', language); in created method